aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncProcessor.java

```diff
@@ -103,69 +105,68 @@ Completable hydrate() {
+                                    new SyncQueriesStartedEvent(modelNames)
+                            )
+                    );
+                })
+                .doOnComplete(() -> {
+                    // When the Completable completes, then emit syncQueriesReady.
+                    Amplify.Hub.publish(HubChannel.DATASTORE,
+                            HubEvent.create(DataStoreChannelEventName.SYNC_QUERIES_READY));
+                });
     }
 
-    private Completable createHydrationTask(Class<? extends Model> modelClass) {
-        ModelSyncMetricsAccumulator metricsAccumulator = new ModelSyncMetricsAccumulator(modelClass);
-        return syncTimeRegistry.lookupLastSyncTime(modelClass)
-            .map(this::filterOutOldSyncTimes)
-            // And for each, perform a sync. The network response will contain an Iterable<ModelWithMetadata<T>>
-            .flatMap(lastSyncTime -> {
-                // Sync all the pages
-                return syncModel(modelClass, lastSyncTime)
-                    // For each ModelWithMetadata, merge it into the local store.
-                    .flatMapCompletable(modelWithMetadata ->
-                        merger.merge(modelWithMetadata, metricsAccumulator::increment)
-                    )
-                    .toSingle(() -> lastSyncTime.exists() ? SyncType.DELTA : SyncType.BASE);
-            })
-            .flatMapCompletable(syncType -> {
-                Completable syncTimeSaveCompletable = SyncType.DELTA.equals(syncType) ?
-                    syncTimeRegistry.saveLastDeltaSyncTime(modelClass, SyncTime.now()) :
-                    syncTimeRegistry.saveLastBaseSyncTime(modelClass, SyncTime.now());
-                return syncTimeSaveCompletable.andThen(Completable.fromAction(() -> {
-                    Amplify.Hub.publish(HubChannel.DATASTORE,
-                                        metricsAccumulator.toModelSyncedEvent(syncType).toHubEvent());
-                }));
-            })
-            .doOnError(failureToSync -> {
-                LOG.warn("Initial cloud sync failed.", failureToSync);
-                DataStoreErrorHandler dataStoreErrorHandler =
-                    dataStoreConfigurationProvider.getConfiguration().getDataStoreErrorHandler();
-                dataStoreErrorHandler.accept(new DataStoreException(
-                    "Initial cloud sync failed.", failureToSync,
-                    "Check your internet connection."
-                ));
-            })
-            .doOnComplete(() ->
-                LOG.info("Successfully sync'd down model state from cloud.")
-            );
+    private Completable createHydrationTask(ModelSchema schema) {
+        ModelSyncMetricsAccumulator metricsAccumulator = new ModelSyncMetricsAccumulator(schema.getName());
+        return syncTimeRegistry.lookupLastSyncTime(schema.getName())
+                .map(this::filterOutOldSyncTimes)
```

Comment:
Please undo format changes

Reply:
These are automated formatting changes by Android Studio. This file seems to have inconsistent formatting as some places the continuation indent is 8 and other its 4 in this file. I've kept it at 8 (and normal indent as 4) while formatting this file. Let me know if I should manually revert these formatting changes.